crates/bevy_core_pipeline/src/auto_exposure/pipeline.rs

```diff
@@ -0,0 +1,94 @@
+use super::compensation_curve::{
+    AutoExposureCompensationCurve, AutoExposureCompensationCurveUniform,
+};
+use bevy_asset::prelude::*;
+use bevy_ecs::prelude::*;
+use bevy_render::{
+    globals::GlobalsUniform,
+    render_resource::{binding_types::*, *},
+    renderer::RenderDevice,
+    texture::Image,
+    view::ViewUniform,
+};
+use std::num::NonZeroU64;
+
+#[derive(Resource)]
+pub struct AutoExposurePipeline {
+    pub histogram_layout: BindGroupLayout,
+    pub histogram_shader: Handle<Shader>,
+}
+
+#[derive(Component)]
+pub struct ViewAutoExposurePipeline {
+    pub histogram_pipeline: CachedComputePipelineId,
+    pub mean_luminance_pipeline: CachedComputePipelineId,
+    pub compensation_curve: Handle<AutoExposureCompensationCurve>,
+    pub metering_mask: Handle<Image>,
+}
+
+#[derive(ShaderType, Clone, Copy)]
+pub struct AutoExposureSettingsUniform {
+    pub(super) min_log_lum: f32,
+    pub(super) inv_log_lum_range: f32,
+    pub(super) log_lum_range: f32,
+    pub(super) low_percent: f32,
+    pub(super) high_percent: f32,
+    pub(super) speed_up: f32,
+    pub(super) speed_down: f32,
+    pub(super) exponential_transition_distance: f32,
+}
+
+#[derive(PartialEq, Eq, Hash, Clone)]
+pub enum Pass {
```

Comment:
We should swap this to a more specific name to make it clearer when searching. `AutoExposurePass` seems fine.

Reply:
Renamed!